src/api/spec/controllers/person_controller_spec.rb

```diff
@@ -106,6 +106,7 @@
           <watchlist>
             <project name="test-proj"/>
             <package name="test-pkg" project="test-proj"/>
+            <request number="1"/>
```

Comment:
@saraycp maybe I overlook something, but for the `PUT` testcase you added the request to the xml but you dont check if its included in the end in the watchlist in line 125

Reply:
shouldn't it also fail in line 126? I think it should include 3 elements in the watchlist, the package, request and project

Reply:
True! I updated the specs to make it more complete.